tests/extras/datasets/holoviews/test_holoviews_writer.py

```diff
@@ -176,7 +176,9 @@ def test_http_filesystem_no_versioning(self):
 
     def test_no_versions(self, versioned_hv_writer):
```

Comment:
Maybe we should rename this and the other test to `test_load_not_supported` instead.

Reply:
Agree, this will be more accurate - will do